console/console-init/ui/package.json

```diff
@@ -7,7 +7,7 @@
     "@apollo/react-hooks": "^3.1.4",
     "@apollo/react-testing": "^3.1.3",
     "@patternfly/react-charts": "^5.3.12",
-    "@patternfly/react-core": "^3.146.0",
+    "@patternfly/react-core": "^3.153.13",
```

Comment:
For which milestone do we expect this change? I ask this because, for 2.0 milestone we have upgraded the patternfly versions to PF 4.

Reply:
I was going to merge to master, then cherry-pick to 0.32.x once 0.32.0 is released.
If this isn't needed on master, I will rebase this on the release-0.32 branch.  But need to wait for 0.32.0 to be released first.

Reply:
I think the PR should be targeting release-0.33, then cherry-pick to release-0.32.

Reply:
I am moving the patternfly/react-core upgrade to a new pr.
The other version upgrades will stay in this PR for master, release-0.33 and release-0.32.